src/tools/qt-gui/src/main.cpp

```diff
@@ -12,15 +12,19 @@
 #include "confignode.hpp"
 #include "undomanager.hpp"
 #include "guibackend.hpp"
+#include "guisettings.hpp"
 
 int main(int argc, char* argv[])
 {
 	QApplication app(argc, argv);
+	//keynames are only split at unescaped slashes
+	app.setProperty("KEY_DELIMITER", QRegularExpression("(?<!\\\\)/"));
```

Comment:
This regex is needed to create a correct hierarchy in the model and has nothing to to with the name of the Key object. It is used to split the QString that contains the keyname to create the new nodes.